src/pages/token-metadata/burn.md

```diff
@@ -42,7 +44,7 @@ await burnV1(umi, {
 
 If the asset that you are trying to burn is part of a collection you additionally need to pass the collectionMetadata account into the function:
 
-{% dialect-switcher title="Burning Assets that are part of a collection" %}
+{% dialect-switcher title="NFT Asset Burn that's part of a collection" %}
```

Comment:
Why this change?  The grammar I think is actually more correct in text being replaced?

wdyt about "Burning NFT Assets that are part of a collection"?